users/api/http/requests.go

```diff
@@ -28,6 +28,18 @@ func (req viewUserInfoReq) validate() error {
 	return nil
 }
 
+type updateUserReq struct {
+	Token string
+	User  users.User
```

Comment:
First, I want to ask why is `Token` public? Second, I still think that here you should have fields that you're updating (for now just `metadata`). If you leave it this way, I can pass email and pass, and the request will be accepted.